packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.Props.ts

```diff
@@ -108,6 +108,16 @@ export interface ISpinButtonProps {
   onDecrement?: (value: string) => string | void;
 
   /**
+   * A callback for when the user put focus on the picker
+   */
+  onFocus?: React.FocusEventHandler<HTMLInputElement>;
```

Comment:
I indeed put the onFocus event handler on an <input> in the render() method.  When user TAB or click, <input> element got the focus and triggers the onFocus event.  So this one has to be of base type <HTMLInputElement>.